cli/roger_build.py

```diff
@@ -82,6 +82,12 @@ def main(self, settingObj, appObj, hooksObj, dockerUtilsObj, dockerObj, args):
             else:
                 repo = data.get('repo', args.app_name)
 
+            build_args = {}
+            if 'build-args' in data:
```

Comment:
No, we are not supporting it via command line, as that needs the user to make sure to pass it every time they build, but if it's in as config, needs to be set once. Also, one doesn't need to maintain caution for env specific values every time they build.